src/tile.cpp

```diff
@@ -927,19 +928,17 @@ void Tile::addThing(int32_t, Thing* thing)
 				if (items) {
 					for (ItemVector::const_iterator it = items->getBeginDownItem(), end = items->getEndDownItem();
 					     it != end; ++it) {
-						MagicField* oldField = (*it)->getMagicField();
+						auto oldField = (*it)->getMagicField();
```

Comment:
Could this be `const auto`
```suggestion
						const auto oldField = (*it)->getMagicField();
```